### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by AnnaVekhina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,16 +3,29 @@
...
+1. Call the [PdfDocumentProcessor.LoadDocument](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.LoadDocument.overloads) method to load a PDF document with an interactive form. 
+2. Call the [PdfDocumentProcessor.GetFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormData) method to retrieve the [PdfFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfFormData) object that contains interactive form data. 
+3. Specify field values of an interactive form. Call the [PdfDocumentProcessor.ApplyFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.ApplyFormData(DevExpress.Pdf.PdfFormData)) method to apply data to the interactive form and save the PDF document.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
3. Specify the interactive form's field values. Call the [PdfDocumentProcessor.ApplyFormData](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.ApplyFormData(DevExpress.Pdf.PdfFormData)) method to apply data to the interactive form and save the PDF document.
```

File: Readme.md [hunk 2]
```diff
@@ -3,16 +3,29 @@
...
+4. You can call the [PdfDocumentProcessor.GetFormFieldNames](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormFieldNames) method to retrieve a list of field names and iterate through the returned string collection.
```
Comment (dirk-pieterse) [inline Readme.md]: It seems strange to start all these sentences with verbs, but then 'You can'?

File: Readme.md [hunk 3]
```diff
@@ -3,16 +3,29 @@
...
+
+Starting with 21.1, we recommend that you use the [PdfDocumentFacade](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade) class to change the PDF document without access to its inner structure. Use the [PdfDocumentFacade.AcroForm](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade.AcroForm) property to get interactive form field options. You can change form fields and appearance properties. Refer to the following example for information on how to use the [PdfDocumentFacade](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade) class: [PDF Document API - How to Change PDF Form Field Parameters](https://github.com/DevExpress-Examples/how-to-change-pdf-form-field-parameters).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
Starting with v21.1, we recommend that you use the [PdfDocumentFacade](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade) class to change the PDF document without access to its inner structure. Use the [PdfDocumentFacade.AcroForm](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade.AcroForm) property to get interactive form field options. You can change form fields and appearance properties. Refer to the following example for information on how to use the [PdfDocumentFacade](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade) class: [PDF Document API - How to Change PDF Form Field Parameters](https://github.com/DevExpress-Examples/how-to-change-pdf-form-field-parameters).
```

`Use the [PdfDocumentFacade.AcroForm] property to get interactive form field options` `You can change form fields and appearance properties` How are these sentences connected? There is no logical flow from one to another.

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 2 above)
Comment (AnnaVekhina) [reply Readme.md]: Can I start this sentence from 'call' as follows: Call the [PdfDocumentProcessor.GetFormFieldNames](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentProcessor.GetFormFieldNames) method to retrieve a list of field names and iterate through the returned string collection.

File: Readme.md (hunk 3 above)
Comment (AnnaVekhina) [reply Readme.md]: I agree with you. Can I rewrite these sentences: 'Use the [PdfDocumentFacade.AcroForm](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Pdf.PdfDocumentFacade.AcroForm) property to get interactive form field options. These options allow you to change form fields and appearance properties.'

File: Readme.md (hunk 2 above)
Comment (dirk-pieterse) [reply Readme.md]: Ok

File: Readme.md (hunk 3 above)
Comment (dirk-pieterse) [reply Readme.md]: Ok

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**